NorthstarDLL/squirrel/squirrel.cpp

```
@@ -395,6 +397,10 @@ void __fastcall ScriptCompileErrorHook(HSquirrelVM* sqvm, const char* error, con
 				R2::Cbuf_AddText(R2::Cbuf_GetCurrentPlayer(), "showconsole", R2::cmd_source_t::kCommandSrcCode);
 		}
 	}
+	else
+	{
+		logger->warn("Not disconnecting, compile error is non-fatal");
```

possibly remove these? not much point and might hide the actual error in logs